cargo-dist/src/config.rs

```diff
@@ -346,6 +353,9 @@ impl DistMetadata {
         if tap.is_none() {
             *tap = workspace_config.tap.clone();
         }
+        if publish_jobs.is_none() {
+            *publish_jobs = workspace_config.publish_jobs.clone();
+        }
```

Comment:
should be a workspace-only field, i think...?

hmm. actually I guess we could make the CI intelligently only publish certain apps to certain venues? if like one only makes sense on pypi or w/e. So I guess package-granularity makes sense.

Reply:
Yeah, it seems like there's room for some extra detail there.